# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/other/DocAccessibleChild.h
# Commit: 4572c1ba670a
# Full Hash: 4572c1ba670a3ddb159cf37ba9684fb6055f260e
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-01-30 05:43:48
# Regressor Bug: 1688972
# File Overlap Count: 1
# Description:
#   Bug 1688972 - Make FocusedChild work across document/process boundaries. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D103102
# ==============================================================================

diff -r f262a5526f6c -r 4572c1ba670a accessible/ipc/other/DocAccessibleChild.h
--- a/accessible/ipc/other/DocAccessibleChild.h	Fri Jan 29 22:44:30 2021 +0200
+++ b/accessible/ipc/other/DocAccessibleChild.h	Fri Jan 29 20:57:30 2021 +0000
@@ -442,9 +442,9 @@
 
   virtual mozilla::ipc::IPCResult RecvTakeFocus(const uint64_t& aID) override;
 
-  virtual mozilla::ipc::IPCResult RecvFocusedChild(const uint64_t& aID,
-                                                   uint64_t* aChild,
-                                                   bool* aOk) override;
+  virtual mozilla::ipc::IPCResult RecvFocusedChild(
+      const uint64_t& aID, PDocAccessibleChild** aResultDoc,
+      uint64_t* aResultID) override;
 
   virtual mozilla::ipc::IPCResult RecvLanguage(const uint64_t& aID,
                                                nsString* aLocale) override;